wiki/Client/Beatmap_editor/Compose/en.md

```diff
@@ -1,351 +1,207 @@
+| Blue | Bookmark |
 
-#### Colour Highlights
+#### Colour sections
 
 | Colour | Description |
 | :-- | :-- |
-| Grey | Break Time |
-| Orange | Kiai Time |
+| Grey | [Break](/wiki/Beatmap/Break) |
+| Orange | [Kiai time](/wiki/Gameplay/Kiai_time) |
 
-#### Bookmarking command
+#### Bookmark actions
 
 | Shortcut | Description |
 | :-- | :-- |
-| `Ctrl` + `B` | Add bookmark at current location. |
-| `Ctrl` + `Shift` + `B` | Remove bookmark at current location. |
-| `Ctrl` + `Right Arrow` | Next bookmark. |
-| `Ctrl` + `Left Arrow` | Previous bookmark. |
-
-At **bottom-right**, you can adjust the **Playback Rate** by **per quarter value.**
+| `Ctrl` + `B` | Add a bookmark at current location. |
+| `Ctrl` + `Shift` + `B` | Remove the bookmark at nearby location. |
+| `Ctrl` + `Right arrow` | Move the current timestamp to the next bookmark. |
+| `Ctrl` + `Left arrow` | Move the current timestamp to the previous bookmark. |
 
 ## Features (osu!mania)
 
-(The top-left hit objects timeline holds no significance. Please ignore it.) For a simple guide: [Basics](https://osu.ppy.sh/community/forums/topics/118868) [Mapping & Keysound](https://osu.ppy.sh/community/forums/topics/139139)
+*See also: [Basics of the osu!mania mapping](https://osu.ppy.sh/community/forums/topics/118868), [osu!mania keysounding](https://osu.ppy.sh/community/forums/topics/139139)*
 
-### Top-Right (Beat Snap Divisor)
+These are the features of osu!mania, in addition to the previous ones.
 
-![Beat Snap Divisor (osu!mania ver.)](img/BEM_BSD.jpg "Beat Snap Divisor (osu!mania ver.)")
+### Beat snap divisor
 
-*Main article: [Beat Snap Divisor](/wiki/Client/Beatmap_editor/Beat_snap_divisor)*
+![](img/beat-snap-divisor-mania.jpg "Beat snap divisor in osu!mania")
 
-**In layman's term, adjusting given length of a coloured sewing thread (timing section) for an embroidery design space (measures, divided by the divisor to be called beats) on the cloth (song).** Since osu!mania highly prioritise every timing of the notes, **Time signature** (MM:B) from the Timing section is added (15:3 = 15th measure 3rd beat) beside the adjustment bar for easier mapper's reference. It can go to negative value (-8:-3 for example) if the first timing section is a *few seconds away* from the start of the beatmap's song file.
+*Main article: [Beat snap divisor](/wiki/Client/Beatmap_editor/Beat_snap_divisor)*
 
-**Insert Break Time** button will put/stop "Break Time" in current timestamp. Beside the button is the **short sound file name storyboarded to the selected(blue) note**, known as **keysound** (refer to "Sampling" section at the end of this page).
+**Time signature** (MM:B) from the timing section is added (15:3 = 15th measure 3rd beat) beside the adjustment bar for easier mapper's reference. It can go to negative value (e.g. -8:-3) if the first timing section is a *few seconds away* from the start of the beatmap's song file.
```

Comment:
this line needs to be removed, since time signature position is a part of the upper timeline, not the beat snap divisor